EventKit/eventkit-cloud
# Proxy formats

# Description of Improvement

Proxy formats are now linked via their own database table rather than through json in Export formats.

## How to test

You can check all tests or the specific test by running:
`docker-compose run --rm eventkit python manage.py test eventkit_cloud.jobs.tests.test_models.TestDataProvider`

## Quality Assurance

The following items have been updated:
